container-images/tcib/base/os/tempest/tempest-extras/tempest-extras.yaml

```diff
@@ -0,0 +1,10 @@
+tcib_actions:
+- run: dnf -y install {{ tcib_packages['common'] | join(' ') }} && dnf clean all && rm -rf /var/cache/dnf
+- run: cd /usr/local; git clone https://opendev.org/openstack/whitebox-tempest-plugin
```

Comment:
whitebox-tempest-plugin is already packaged in rdo: https://github.com/rdo-packages/whitebox-tempest-plugin-distgit Do we still want to use whitebox-tempest-plugin here?

Reply:
Yes compute is definitely interested in finding a way to incorporate the whitebox plugin into a tempest container.  Either with this approach or if we can find a way in the build process to add a whitebox (or some other plugin) layer to the container that we can reference when testing.

Reply:
Thank you @jamepark4 , Let's give it a go.